analysis_options.yaml

```diff
@@ -8,6 +8,8 @@ analyzer:
     unused_local_variable: error
     dead_code: error
     deprecated_member_use_from_same_package: ignore
+    lines_longer_than_80_chars: ignore
+    avoid_dynamic_calls: ignore
```

Comment:
We can do that outside this PR, but maybe we should just adopt `dart_flutter_team_lints` without any custom options.

Reply:
I agree, but there are a bunch of violations, so better to split that up into separate PRs.